### ISSUE TITLE
Broken backup when shutting down computer while backup is running

### ISSUE THREAD

**Issue by David-Pa** [duplicati/duplicati#1252]
This is probably a duplicate, so feel free to merge it.
Duplicati 2.0.0.73, Windows 8.1 running from the Task Scheduler with elevated privileges.

I've set quite a large backup using a remote backend, which would take a few days to complete. I cannot keep the computer on during all this time, so it's obvious that the backup has to be paused and resumed a few times.

After shutting down the computer without pausing/canceling the backup manually, the next time Duplicati tried to resume it, it threw errors about files that need to be deleted from the server which cannot be deleted because... they do not exist! :)

As first I thought it was funny, but Duplicati wouldn't proceed. So I had to manually create those files on the backend, and only then Duplicati was able to delete them and proceed with the backup.

Unfortunately, the second time it happened, it wouldn't continue even when I created the files on the server. The only way it would proceed was to completely delete the local database, and then run "Repair local database" (running the repair without first deleting was no good). I also had to stop the Duplicati service and delete the database manually, but I've opened a separate ticket for this issue. Needless to say that downloading multiple gigabytes of data took a whole lot of time.

So to summarize the story, here are the key issues I encountered:
1. Backups should not break when shutting down the computer or otherwise interrupted.
2. Duplicati should not break operation on failure to delete a file, if the failure reason is that the file does not exist.
3. Repair database should work even without deleting the database first.

Thank you!
##

**Comment by rocifier** [comment]
This seems like a really important issue. When I tested this, I didn't experience problems to this extent, but I did experience the backup taking a long time to resume. It had to go through my 200k files all over again and start uploading something for about 20 minutes, only then did the calculation resume from where it left off (about 4 gigabytes of uploads already done). I guess it depends what it's doing when you interrupt it?

**Comment by kenkendk** [comment]
This is related to #1243, but not the same.

The problem with the delete of non-existing files is fixed and will be part of the next preview release.
